db/migrate/20150929142405_change_postalcode_format_in_venues.rb

```diff
@@ -0,0 +1,9 @@
+class ChangePostalcodeFormatInVenues < ActiveRecord::Migration
+  def up
+    change_column :venues, :postalcode, :string
+  end
+
+  def down
+    change_column :venues, :postalcode, :integer
```

Comment:
This won't work. If the field already has a string value, turning that field into integer type raises an error.
If we force going back to integer type, we need to alter/delete the existing postalcode (or at least its non number parts). Otherwise this is an irreversible migration.

@hennevogel your input? Reverse the migration by deleting non number chars or make it irreversible?